src/js/components/bulkDownload/BulkDownloadPage.jsx

```diff
@@ -67,6 +69,7 @@ export default class BulkDownloadPage extends React.Component {
 
     changeDataType(dataType) {
         this.props.setDataType(dataType);
+        Router.history.replace('/bulk_download');
```

Comment:
See later comment about avoiding spreading routing logic around.

Reply:
This removes `/award_data_archive` from the url when a user clicks an item in the sidebar